ta/ta.mk

```diff
@@ -29,6 +34,9 @@ ta-mk-file-export-vars-$(sm) += CFG_SECURE_DATA_PATH
 ta-mk-file-export-vars-$(sm) += CFG_TA_MBEDTLS_SELF_TEST
 ta-mk-file-export-vars-$(sm) += CFG_TA_MBEDTLS
 ta-mk-file-export-vars-$(sm) += CFG_TA_MBEDTLS_MPI
+ta-mk-file-export-vars-$(sm) += CFG_TA_STACK_PROTECTOR
+ta-mk-file-export-vars-$(sm) += CFG_TA_STACK_PROTECTOR_STRONG
+ta-mk-file-export-vars-$(sm) += CFG_TA_STACK_PROTECTOR_ALL
```

Comment:
Not sure these need to be exported. They change the TA compilation flags, so I think it should be left to the user to set or not some of these when building the TA (assuming objects with and without stack protector can be mixed?).

Reply:
CFG_TA_STACK_PROTECTOR affects the library. It should be exported for TA to know wether or not the lib in the devkit embeds stack protection.

Reply:
Why would you need to know unless it has an impact on how you build your TA? This information is available in the generated OP-TEE config file anyways (`<TA dev kit>/host_include/conf.mk`)

Reply:
okay I removed the TA stack protector config export